### DISCUSSION TITLE
Support for prisma-client-py / How to run commands after pip installation but before deployment?

### DISCUSSION THREAD

**Discussion by tchanxx** [UnitedIncome/serverless-python-requirements#782] [Q&A]
Hi, I am trying to use [prisma-client-py](https://github.com/RobertCraigie/prisma-client-py) with the Serverless Framework. To use the Prisma Client, a command `prisma generate` needs to be run after the pip installation occurs so that the proper Prisma Python Client is generated in the environment. I cannot simply copy-and-paste the generated client because it changes based on the environment the client is generated in (my local is MacOS but I want to deploy to Arm64). 

Are there any tricks to run a command after the pip installation step? I read the docs and I see instructions on adding instructions to the Docker Build and the Docker Run commands but I don't know if those are helpful.

**Comment by g-gregoire** [comment]
could also use help with this.. any luck?
